### ISSUE TITLE
macOS (new?) issue

### ISSUE THREAD

**Issue by sh4tteredd** [berdav/greenpass#43]
Hi,
I used to use this utility and it worked great! But know on the exact machine (latest 12.3 arm64 M1), probably after an update, it don't work anymore. I tried with different DGC, same result

<img width="627" alt="Screenshot 2022-03-30 at 15 33 24" src="https://user-images.githubusercontent.com/55893559/160847345-4b218fb2-f574-47a6-bcce-4f3c4b7d9393.png">

Is this a know issue on mac? Or what

Thanks!

**Comment by egglessness** [comment]
Yes, I can confirm the issue in macOS 12.4. This is due to the fact that locale environment variables are improperly populated.

As a workaround, you can set the following environment variables before firing up the script:
```
export LC_ALL=en_US.UTF-8  
export LANG=en_US.UTF-8
```

I think the script should fallback to `en_US` when the `locale.getdefaultlocale()[0]` returns `None`.
